### PR TITLE
Code style fixes

### PR DESCRIPTION
**Pull request by ITaluone**
This PR solves some Visual Studio errors

## IMPORTANT 

* [x] The code complies with the [Coding Guidelines for C#](https://www.csharpcodingguidelines.com/).
* [ ] The changes are covered by unit tests which follow the Arrange-Act-Assert syntax and the naming conventions such as is used [in these tests](../tree/develop/Tests/FluentAssertions.Equivalency.Specs/MemberMatchingSpecs.cs#L51-L430).
* [ ] If the PR adds a feature or fixes a bug, please update [the release notes](../tree/develop/docs/_pages/releases.md) with a functional description that explains what the change means to consumers of this library, which are published on the [website](https://fluentassertions.com/releases).
* [ ] If the PR changes the public API the changes needs to be included by running [AcceptApiChanges.ps1](../tree/develop/AcceptApiChanges.ps1) or [AcceptApiChanges.sh](../tree/develop/AcceptApiChanges.sh).
* [ ] If the PR affects [the documentation](../tree/develop/docs/_pages), please include your changes in this pull request so the documentation will appear on the [website](https://www.fluentassertions.com/introduction).

### CODE REVIEW

File: Tests/Benchmarks/UsersOfGetClosedGenericInterfaces.cs [hunk 1]
```diff
@@ -166,7 +165,7 @@ public void GlobalSetup()
             }
 
             context = new Context();
-            config = new Config();
+            new Config();
```
Comment (jnyrup) [inline Tests/Benchmarks/UsersOfGetClosedGenericInterfaces.cs:168]: Have a look at the changes in 684b5782b79d0f50094e61451dc77eaa5bc27647

`IEquivalencyAssertionOptions` is now a member of `IEquivalencyValidationContext`.
So I would just make `Context.Options` writable and initialize it as
```cs
context = new Context()
{
    Options = new Config()
};
```

**Review by jnyrup — CHANGES_REQUESTED** [review]

**Comment by ITaluone** [comment]
Or initialize it like: `public IEquivalencyAssertionOptions Options { get; } = new Config();` ?

**Comment by jnyrup** [comment]
> Or initialize it like: `public IEquivalencyAssertionOptions Options { get; } = new Config();` ?

No need to spend more energy on this class.
These benchmarks are not run in any automatic way.

**Comment by ITaluone** [comment]
Just realized there were changes in wrong commits :)

**Pull request merged**